process2.cpp

```diff
@@ -0,0 +1,138 @@
+#include<iostream>
+//#include<sys/types.h>
+#include<netinet/in.h>
+#include<unistd.h>
+#include<sys/socket.h>
+#include<sys/ioctl.h>
+#include<sys/epoll.h>
+#include<arpa/inet.h>
+#include<vector>
+#include<string.h>
+//#include<sys/socketvar.h>
+//#include<fcntl.h>
+#define SERV_PORT 8000
+int main()
```

Comment:
Hi @thanhthai97 ,
Thanks for your efforts. Overall, the code is good.
To make the program like the real project we need handle signal e.g when user type "CTRL + C" your program can listen and handle this signal.
Refer this example:
https://github.com/mtzguido/epoll/blob/master/signalfd.c

Thanks,
Sang Vu